packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -224,6 +232,10 @@ export class Cluster extends Resource implements ICluster {
       physicalName: props.clusterName,
     });
 
+    if ((props.containerInsights !== undefined) && props.containerInsightsV2) {
+      throw new Error('You can only specify either containerInsights or containerInsightsLevel. Alternatively, you can leave both blank');
```

Comment:
More straightforward wording

```suggestion
      throw new Error('You cannot set both containerInsights and containerInsightsV2');
```

Reply:
Sounds good. Updated.